great_expectations/execution_engine/split_and_sample/sqlalchemy_data_splitter.py

```diff
@@ -526,19 +526,18 @@ def get_split_query_for_data_for_batch_identifiers_for_split_on_date_parts(
                         )
                     )
 
-                concat_clause: List[Label] = [
+                concat_clause = [
```

Comment:
Shouldn't we declare this type above the `if`/`else` block? If i'm reading this code I'm going to skip the `if` block, get to this variable, then need to scan above and look in the unrelated code path to find the type.

Reply:
That makes sense to me.

Reply:
Done